indexer/SymbolFormatter.cc

```diff
@@ -33,8 +137,18 @@ std::string_view SymbolFormatter::getMacroSymbol(clang::SourceLocation defLoc) {
   } else {
     filename = std::string_view(defPLoc.getFilename());
   }
-  std::string out{fmt::format("c . todo-pkg todo-version {}:{}:{}#", filename,
-                              defPLoc.getLine(), defPLoc.getColumn())};
+
+  // Technically, ':' is used by SCIP for <meta>, but using ':'
+  // here lines up with other situations like compiler errors.
+  auto name = this->formatTemporary("{}:{}:{}", filename, defPLoc.getLine(),
```

Comment:
One slightly annoying thing here, the simplest way to do this generates uglier symbols with escaping backticks. If instead we split this into 3 different descriptors, then the first two would be `Meta` (to get the `:`) and the last one would be a `Macro`. So there wouldn't be surrounding backticks. However, the current API requires `string_view` names for the descriptors, but we shouldn't allocate temporary strings in between.